processing/bloom_handler.go

```diff
@@ -242,7 +243,12 @@ func (a *BloomHandler) Consume(e *types.Entry) error {
 	if e.EventType == "dns" {
 		a.Lock()
 		if a.IocBloom.Check([]byte(e.DNSRRName)) {
-			n := MakeAlertEntryForHit(*e, "dns", a.AlertPrefix, e.DNSRRName)
+			var n types.Entry
+			if e.DNSType == "query" {
+				n = MakeAlertEntryForHit(*e, "dns-req", a.AlertPrefix, e.DNSRRName)
+			} else {
```

Comment:
I would use another match here for `dns-resp` and a final else to log malformed/different types?

Reply:
The idea here was to always have a defined value, with 'answer' being the default. But it surely can't hurt to be more verbose. Update to follow!